core/src/main/java/kafka/server/share/SharePartition.java

```diff
@@ -219,11 +220,16 @@ public static RecordState forId(byte id) {
      */
     private final int maxDeliveryCount;
 
+    /**
+     * The group config manager is used to retrieve the values for dynamic group configurations
+     */
+    private final GroupConfigManager groupConfigManager;
+
     /**
      * The record lock duration is used to limit the duration for which a consumer can acquire a record.
```

Comment:
This comment should say that this is the default which is used unless the group has a configuration which overrides it.